create_graphics.py

```diff
@@ -531,7 +539,7 @@ def parallel_skewt(cla, fhr, ds, site, workdir):
     plt.savefig(
         png_path,
         bbox_inches='tight',
-        dpi='figure',
+        dpi=cla.img_res,
```

Comment:
I think this would be fine for a line graph (default = 72), but would you mind providing examples in the PR of the before and after, along with the change in size of the resulting PNG files? I have no clue what the 'figure' dpi would be for these images.

Reply:
That's a good question. Has anyone run the skew-t side of this yet? I have a retro sitting around that I will test this on, but as of right now, I do not know. I have never used 'figure' before as a value for dpi in savefig.

Reply:
I'm not sure what you mean. We run this in real-time runs constantly. I just don't know what the dpi numerical value would be for a SkewT figure. I'd like to make sure that 72 as a default wouldn't degrade what is currently produce. My feeling is that it wouldn't, but we do need to check.

Reply:
Okay, so I have looked at this, and the results are pleasing.

Here is a sample skew-t plot (Denver site) using the existing code (DPI = 72); image size = 211 KB:
![DNR_72469_skewt_f001](https://user-images.githubusercontent.com/31049415/160935998-ad2f0489-e39e-486d-988e-c2dc9837d824.png)

And here is the same image using the updated code with DPI = 150 (image size = 555 KB):
![DNR_skewt_r150](https://user-images.githubusercontent.com/31049415/160936001-4a1f705a-c0b7-4fe1-9997-d565f48298c3.png)

"DPI" = "dots per inch" and is a generic characteristic of making images. There is nothing special about the difference between dpi for skewts and dpi for horizontal cross sections; the same value should work just fine for both types of images.

Regarding 'figure' as the value of dpi in savefig, the documentation for matplotlib says:

>     The resolution in dots per inch. If 'figure', use the figure's dpi value.

It turns out that 'dpi' is also a kwarg for pyplot.figure when declaring a new figure. I was not aware of this previously. However, I don't see dpi explicitly set in skewt.py at line 401:

`        fig = plt.figure(figsize=(12, 12))`

So I presume some default value is being used there. I just figured I would change the code for consistency's sake. I can confirm that if 
`dpi='figure'`
is used, then -r is ignored. So I think changing this in both places makes the code more consistent.

Reply:
I think this looks totally acceptable, and am happy with the outcome. The text is slightly blurry with the dpi=72 option, but if we get complaints, we can probably just add the appropriate CLA to the workflows that triggered the complaints. I don't think this will break anything.